src/CallHandler.tsx

```diff
@@ -98,6 +97,18 @@ export enum PlaceCallType {
     ScreenSharing = 'screensharing',
 }
 
+function getRemoteAudioElement(): HTMLAudioElement {
+    // this needs to be somewhere at the top of the DOM which
+    // always exists to avoid audio interruptions.
+    // Might as well just use DOM.
+    const remoteAudioElement = document.getElementById("remoteAudio") as HTMLAudioElement;
+    if (!remoteAudioElement) {
+        console.error("Failed to find remoteAudio element - cannot play audio!"
+            + "You need to add an <audio/> to the DOM.");
```

Comment:
it looks like a bit of an accident that this falls through - can we get an explicit `return null;` to imply the contract when we come back to maintain it?